### ISSUE TITLE
When installed then using common apps writing latvian letter ū then it doesnt write but opens Multipass and cant use computer

### ISSUE THREAD

**Issue by kangarooo** [canonical/multipass#3300]
**Describe the bug**
Describe what your problem is.
When installed then using common apps writing latvian letter ū then it doesnt write but opens Multipass and cant use computer
**To Reproduce**
How, and what happened?
1. `multipass` installed
2. write letter ū in notepad with alt right + u
3. doesnt writes, but opens multipass

**Expected behavior**
What did you expect to happen?
Cant write anymore in Latvian

**Logs**
Please provide logs from the daemon, see [accessing logs](https://multipass.run/docs/accessing-logs) on where to find them on your platform.

**Additional info**
 - Windows 10
- `multipass version` yesterdays version
- `multipass info --all`
- `multipass get local.driver`

**Additional context**

**Comment by ricab** [comment]
Hi @kangarooo, thanks for reporting. I was able to reproduce this. The Multipass GUI has a hotkey to open a shell into the primary instance. By default, that's "Ctrl+Alt+U", but for some reason "AltGr+U" triggers it too when the Latvian language is selected.

You can work around this by setting a [different hotkey](https://multipass.run/docs/gui-hotkey) with `multipass set client.gui.hotkey=<key-combo>`. For example you could set it to <kbd>Ctrl</kbd>+<kbd>.</kbd> with `multipass set client.gui.hotkey=Ctrl+.`. You can also disable the hotkey entirely: `multipass set client.gui.hotkey=`.

**Comment by ricab** [comment]
After looking a little deeper, I found that <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>u</kbd> also results in ū with the Latvian keyboard. 

I am not sure that there is much we can do about this. We rely on the text representation for the hotkey setting to work well across platforms. We have tried to pick a default that will work well for most people, but we can't account for all clashes with all input schemes and all other applications. Hence the setting: everyone can set a hotkey to their liking :slightly_smiling_face:
